### DISCUSSION TITLE
Discord server

### DISCUSSION THREAD

**Discussion by urchade** [urchade/GLiNER#75] [Announcements]
I've launched a dedicated Discord server for GLiNER. Join our community here: https://discord.gg/Y2yVxpSQnG

**Comment by daniel-scalioni** [comment]
I tryed to enter on this community, but its not avaible.
Can u help me?

**Comment by manickavela29** [comment]
yes @urchade , the link seems to have been expired,
can you share a new link hear

**Reply by manickavela29** [reply]
ok, it is working,
we just have to login to discuss before trying the link
